client/src/components/ViewProfileDetail.js

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { Card } from 'react-bootstrap';
+import { ProfileContext } from '../context/ProfileContext';
+import avatar from '../assets/icons/avatar.svg';
+import close from '../assets/icons/IconX.svg';
+
+const ViewProfileDetail = () => {
+	const { profile, clearProfile } = useContext(ProfileContext);
+	console.log('prof', profile);
+	const handleClick = ()=>{
+		clearProfile();
+	};
+
+	return (
+		<Container>
+  			<Icon variant="top" src={close} onClick={handleClick} />
```

Comment:
Also consider, 
```suggestion
  			<Icon variant="top" src={close} onClick={clearProfile} />
```
simplifies this somewhat?